topics/blockchain/index.md

```diff
@@ -0,0 +1,10 @@
+---
+created_by: Satoshi Nakamoto
+display_name: Blockchain
+released: '2008'
+short_description:  Blockchain technology provides a method to make electronic payments without needing any financial institute as banks.
```

Comment:
> without needing any financial institute as banks

This part reads a little strangely. What about "without the need for a trusted authority"?